src/csharp/Directory.Build.props

```diff
@@ -34,6 +34,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Content Include="$(MSBuildThisFileDirectory)..\..\LICENSE"
```

Comment:
I wonder if we should change the `PackageLicenseExpression` to `<PackageLicenseFile>LICENSE</PackageLicenseFile>` instead.

That way there is only 1 "truth" or "master" license - the file that is in the root of the repo. If we ever changed that file (to say Apache 2.0), the nuget package would say it is `MIT`, but this LICENSE file would say `Apache.

Reply:
fixed.